app/rubin-2025/register-form.jsx

```diff
@@ -79,233 +79,294 @@ class RegisterForm extends React.Component {
+              )}
+            </div>
+
             {"nameConflict" in this.state.pending && (
               <LoadingIndicator />
             )}
-            {nameConflict && (
-              <span className="form-help error">
-                <Translate content="registerForm.nameConflict" />{' '}
-                <a href={`${window.location.origin}/reset-password`}>
-                  <Translate content="registerForm.forgotPassword" />
-                </a>
-              </span>
-            )}
+
+            <div aria-live="polite" id="register-form-login-help-message-2">
+              {nameConflict && (
+                <span className="form-help error">
+                  <Translate content="registerForm.nameConflict" />{' '}
+                  <a href={`${window.location.origin}/reset-password`}>
+                    <Translate content="registerForm.forgotPassword" />
+                  </a>
+                </span>
+              )}
+            </div>
+
             {(this.state.input_login?.length > 0
               && nameConflict === false
+              && !("nameConflict" in this.state.pending)
             ) && (
               <span className="form-help success">
                 <Translate content="registerForm.looksGood" />
               </span>
             )}
+
             <Translate className="form-help info right-align" content="registerForm.required" />
-          </span>
+          </div>
+
           <input
-            type="text"
-            name="login"
-            id="register-form-login"
-            value={this.state.input_login}
+            aria-describedby="register-form-error-message register-form-login-help-message-1 register-form-login-help-message-2"
             className="standard-input full"
             disabled={inputDisabled}
-            autoFocus
-            onChange={this.handleUserInput}
+            id="register-form-login"
             maxLength="255"
+            name="login"
+            onChange={this.handleUserInput}
+            type="text"
+            value={this.state.input_login}
           />
-          <Translate component="span" className="form-help info" content="registerForm.whyUserName" />
-          {this.state.underAge && (
-            <Translate component="span" className="form-help info" content="registerForm.notRealName" />
-          )}
-        </label>
 
-        <br />
-
-        <label>
-          <span className="columns-container inline spread">
-            <Translate content="registerForm.password" />
-            {passwordTooShort && (
-              <Translate className="form-help error" content="registerForm.passwordTooShort" />
+          <div>
+            <Translate component="span" className="form-help info" content="registerForm.whyUserName" />
+            {this.state.underAge && (
+              <Translate component="span" className="form-help info" content="registerForm.notRealName" />
             )}
+          </div>
```

Comment:
Screenreaders in forms mode will ignore this `<div>` (it's not a form element) so this might need to be added to the field's accessible description too.